docs/CNVs/CNV-methods.tex

```diff
@@ -670,125 +712,170 @@ \subsection{Generative Model for Read Counts} \label{coverage-model}
 
 \item Use an algorithm that does not waste time calculating all principal components when we only want the few most significant ones.
 
+\item Make a universal panel of normals for both sexes by taking into account targets on sex chromosomes on par with autosomal targets.
+
+\item A reliable method for detecting and excluding outlier samples from the panel of normals, in particular, those with abnormal ploidy.
```

Comment:
nitpicking, but perhaps rephrase this as an action to be consistent with the other goals listed ("connect", "take", "choose", "use", "make")